openedx/features/discounts/applicability.py

```diff
@@ -147,8 +147,6 @@ def _is_in_holdback_and_bucket(user):
     Return whether the specified user is in the first-purchase-discount holdback group.
     This will also stable bucket the user.
     """
-    if datetime(2020, 8, 1, tzinfo=pytz.UTC) <= datetime.now(tz=pytz.UTC):
```

Comment:
not quite sure why this date was hardcoded here but now it seems to be causing the test failure.

Reply:
A recent PR skipped the test due to this, might want to undo that also: https://github.com/edx/edx-platform/commit/f9a5b7790b879ee69435d844ed5e8c2aac964999#diff-48bc2d491a7bebd89e7a09bc6d4e4771R169

Reply:
done